website/content/docs/v0.9/Guides/editing-machine-configuration.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Editing Machine Configuration"
+description: "How to edit and patch Talos machine configuration, with reboot, immediately, or stage update on reboot."
+---
+
+Talos node state is fully defined by [machine configuration](../Reference/configuration/).
+Initial configuration is delivered to the node at bootstrap time, but configuration can be updated while the node is running.
+
+> Note: Be sure that config is persisted so that configuration updates are not overwritten on reboots.
+> Configuration persistence was enabled by default since Talos 0.5 (`persist: true` in machine configuration).
+
+There are three `talosctl` commands which facilitate machine configuration updates:
+
+* `talosctl apply-config` to apply configuration from the file
+* `talosctl edit machineconfig` to launch an editor with existing node configuration, make changes and apply configuration back
+* `talosctl patch machineconfig` to apply automated machine configuration via JSON patch
```

Comment:
I think it also makes sense to specify environment variables that allow changing the editor.
`EDITOR` and `TALOS_EDITOR`.